docs/packages/markdown/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@material-ui/markdown",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

A use a single version for simplicity, as much as we can. 

```suggestion
  "version": "5.0.0-beta.0",
```

---

don't do that. it's not associated with the other packages. I don't think it even changed.

---

We did fix a bug on windows. Patch bump is ok. It should not go to 5.x beta. That doesn't make any sense and is just confusing.

---

@eps1lon Could we expand on the pros and cons of each approach?

From my perspective on this one:

1. it's not independent, it's tailored for the need of the docs, it's private, docs are in beta.0
2. in the best possible world, we won't give this module a version, it's a subfolder that supports the docs, like the other.